Description: Change shell command from node to nodejs
Author: Jonas Smedegaard <dr@jones.dk>
 David Paleino <dapal@debian.org>
Bug-Debian: http://bugs.debian.org/614907
Last-Update: 2012-09-20

--- a/doc/node.1
+++ b/doc/node.1
@@ -2,12 +2,12 @@
 
 
 .SH "NAME"
-node \- Server-side JavaScript
+nodejs \- Server-side JavaScript
 
 .SH SYNOPSIS
 
 
-.B node
+.B nodejs
 [
 .B \-v
 ]
--- a/tools/wafadmin/Tools/node_addon.py
+++ b/tools/wafadmin/Tools/node_addon.py
@@ -49,7 +49,7 @@ def detect(conf):
   found = os.path.exists(conf.env['NODE_PATH'])
   conf.check_message('node path', '', found, conf.env['NODE_PATH'])
 
-  found = os.path.exists(join(prefix, 'bin', 'node'))
+  found = os.path.exists(join(prefix, 'bin', 'nodejs'))
   conf.check_message('node prefix', '', found, prefix)
 
   ## On Cygwin we need to link to the generated symbol definitions
